independent-publisher-connectors/eBay/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "AmitShar-Artesian-SBX-f1c9d9c1e-a4e1f868",
```

Comment:
@artesianst secrets look alike values are not allowed. plese replace with dummy data.